repo-scripts/size-analysis/bundle-definitions/app-check.json

```diff
@@ -0,0 +1,92 @@
+[
+  {
+    "name": "ReCaptchaV3Provider",
+    "dependencies": [
+      {
+        "packageName": "firebase",
+        "versionOrTag": "exp",
+        "imports": [
+          {
+            "path": "app",
+            "imports": [
+              "initializeApp"
+            ]
+          }
+        ]
+      },
+      {
+        "packageName": "firebase",
+        "versionOrTag": "exp",
+        "imports": [
+          {
+            "path": "app-check",
+            "imports": [
+              "initializeAppCheck",
```

Comment:
Did these used to include `getToken`? I thought I remembered seeing them. It makes sense because if app check is being used, getToken() will eventually be called, if not by the user then by another library (storage, functions, etc).

Reply:
Yes, it was there before. I got confused this morning when reading the [code sample](https://firebase.google.com/docs/app-check/web/recaptcha-enterprise-provider#initialize) for App Check Enterprise, which doesn't reference `getToken`, and I wondered why it was there. I forgot that it was actually Feiyang who added `getToken` for me in the bundle doc last week. I was actually about to ask you, but got stuck in some changeset warnings in this pull request.

Anyway, I have added it back both in the bundle definition doc and the json files in this pull request. Please take a look. Thanks!